### DISCUSSION TITLE
Help! Change username

### DISCUSSION THREAD

**Discussion by annicasivaa** [community/community#77416] [New to GitHub]
Hi,
I recently had to delete a Github account under the name annicasiva and make a new account which is this: annica1910.
I cant seem to change the username now even though it has been months, and I can't even make a new account with that username.
I have applied for jobs using github link to that profile in my CV, and would really like to upload files under that username.
Can you please help me with this?

**Comment by yagodorea** [comment]
I'm having the same issue (different context though). I saw in [GH docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account) that it takes 90 days for the username to be released, I opened a ticket to try to speed up the process but they seem to be **very** slow to respond :/

**Reply by devkhishan** [reply]
Brother I have 2 accounts, and I need to change the username of account 2 to account 1.  If I update the username of account 2 to something else, Can I use the username of account 2 on my account 1 ? Any Ideas ?

**Reply by 567jin** [reply]
I have the same problem, but I have forgotten the password email for one of my accounts. This should work for you.
